Common/Data/Models/MobData.cs

```diff
@@ -6,4 +6,25 @@ public class MobData
 {
 	public int NetId { get; set; }
 	public List<MobEntry> Entries { get; set; }
+}
+
+public class MobEntry
+{
+	public float? Scale { get; set; }
+	public string Prefix { get; set; }
+	public decimal Weight { get; set; }
+	public MobStats Stats { get; set; }
+	public List<MobEntryAffix> Affixes { get; set; }
+	public string Requirements { get; set; }
+}
+
+public class MobEntryAffix
+{
+	public string Name { get; set; }
```

Comment:
Do we really need a wrapper around a single `string`? The only substantial difference this would make is making access `Affixes[0].Name` instead of `Affixes[0]` and maybe a little more documentation.

Reply:
I tried to make the structure as scalable as possible. While it's just one property right now, it is likely to scale. Such as affix multipliers.